_posts/tumblr/2015-04-27-learning-git-with-mario-bros.md

```diff
@@ -0,0 +1,34 @@
+---
+layout:     post
+title:      Learning Git with Mario Bros
+date:       2015-04-27 09:41:29
+summary:    Like playing Mario Bros? Here's one programming lesson hidden in the famous videogame.
+categories: code
+author:     stefy
+---
+![mariobros](/images/mariobros.jpg)
+
+
+###What Mario Bros and GitHub have in common:
+
+Part of learning how to program is understanding how GitHub works. Github is an open source project started by Linux creator Linus Torvalds. It basically allows you to make changes to a project by copying the repository to your own system. Then you can make your changes on your local copy and “check them in” to the central server. When I had trouble putting this to practice, Max, our awesome, back-end lead, came up with this brilliant, yet simple, explanation:
```

Comment:
You probably mean Git, not Github. Git was made by Linus Torvalds, and is open source. Github itself is closed-source and a company.

Y gracias, I feel humbled :blush: